### ISSUE TITLE
train

### ISSUE THREAD

**Issue by Tanghui2000** [Zhendong-Wang/Prompt-Diffusion#10]
Hello, thank you for your work, which is enlightening to me.
I have some questions to consult. I tried to train grayscale images and color images as a set of "examples", so as to achieve image coloring. However, the generated images are very different from the original ones. 
Why does this happen? How to ensure consistency with the original image?

**Comment by Zhendong-Wang** [comment]
The consistency is maintained because the model is also given the query image as the inputs. As we show in the Figure 3. The training also depends on the dataset, you input and output image pairs should be consistent.

**Comment by Tanghui2000** [comment]
> The consistency is maintained because the model is also given the query image as the inputs. As we show in the Figure 3. The training also depends on the dataset, you input and output image pairs should be consistent.

Thank you for helping me. It is now possible to keep the input consistent with the content of the generated image. But the resulting image's contrast and saturation seem to have changed, making it seem more noisy than the original, which bothers me. I'm using the coco dataset. Some of the images in the coco dataset are noisy. Is this misleading? Or is it something else?
![屏幕截图 2023-11-29 091525](https://github.com/Zhendong-Wang/Prompt-Diffusion/assets/132808352/216db08b-ea0a-431a-913a-06cb4246a4db)

**Comment by Zhendong-Wang** [comment]
Yes, here I think image quality is important for finetuning. If the dataset is noisy, you may need longer training to converge and may converge to worse results.
